packages/flame_oxygen/lib/src/system/base_system.dart

```diff
@@ -31,28 +41,24 @@ class FlameSystem extends System with RenderSystem, UpdateSystem {
         final angle = entity.get<AngleComponent>()!.radians;
         canvas.rotate(angle);
       }
-      // final delta = -anchor.toVector2()
-      //   ..multiply(size);
-      // canvas.translate(delta.x, delta.y);
+      final delta = -anchor.toVector2()
+        ..multiply(size);
+      canvas.translate(delta.x, delta.y);
 
       // Handle inverted rendering by moving center and flipping.
       // if (renderFlipX || renderFlipY) {
       //   canvas.translate(width / 2, height / 2);
       //   canvas.scale(renderFlipX ? -1.0 : 1.0, renderFlipY ? -1.0 : 1.0);
       //   canvas.translate(-width / 2, -height / 2);
       // }
```

Comment:
I guess this should be removed?

Reply:
Ah no, that is still something i want to implement as the `PositionComponent` from FCS also has it. I just haven't thought of a clean way to do that. Still noodling on it :D

Reply:
Still noodling on this I guess? :smile:

Reply:
Ah I forgot about this! Will have a look at it tomorrow!

Reply:
Tomorrow was today, noodling done